src/main/java/games/strategy/triplea/ai/proAI/ProAi.java

```diff
@@ -55,19 +55,19 @@
 /**
  * Pro AI.
  */
-public class ProAI extends AbstractAI {
+public class ProAi extends AbstractAi {
 
   // Odds calculator
-  private static final IOddsCalculator concurrentCalc = new ConcurrentOddsCalculator("ProAI");
+  private static final IOddsCalculator concurrentCalc = new ConcurrentOddsCalculator("ProAi");
```

Comment:
This string is simply used as the prefix for the name of the threads created by `ConcurrentOddsCalculator`.  It does not appear to be referenced by any other code.